index.js

```diff
@@ -3,11 +3,14 @@ const ManyKeysMap = require('many-keys-map');
 const pDefer = require('p-defer');
 
 const cache = new ManyKeysMap();
-const isDomReady = () => document.readyState === 'interactive' || document.readyState === 'complete';
+const isDomReady = target =>
+	(target.ownerDocument || target).readyState === 'interactive' ||
+	(target.ownerDocument || target).readyState === 'complete';
```

Comment:
`document.ownerDocument === undefined`, so `|| target` is necessary when `target === document`